devicemodel/hw/platform/acpi/acpi_parser.c

```diff
@@ -0,0 +1,150 @@
+/*-
+ * Copyright (c) 2011 NetApp, Inc.
```

Comment:
Please see and answer my question from the mailing list.

Reply:
OK

Reply:
I will change it to 'Copyright (C) 2021 Intel Corporation. '